idunn/api/places.py

```diff
@@ -31,18 +32,26 @@ def validate_lang(lang):
     return lang.lower()
 
 
-def log_custom_headers(id, headers: Headers):
-    custom_data = {}
+def log_place_request(place: BasePlace, headers: Headers):
+    custom_data = {
+        'id': place.get_id(),
+        'name': place.get_local_name(),
+        'class': place.get_class_name(),
+        'subclass': place.get_subclass_name(),
```

Comment:
I would use `class_name` and `subclass_name` to be consistent with the API response